fili-wikipedia-example/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM druidio/example-cluster:latest
+LABEL maintainer="https://groups.google.com/forum/#!forum/fili-developers"
+
+# fili github information
+ENV GITHUB_OWNER=yahoo BRANCH=master
+
+RUN apt-get update && apt-get install -y --no-install-recommends git
```

Comment:
I could add 
```
RUN apt-get install -y curl
```
before the chmod in the Dockerfile. Is there a better way to do this?